homeassistant/components/niko_home_control/const.py

```diff
@@ -4,3 +4,18 @@
 
 DOMAIN = "niko_home_control"
 _LOGGER = logging.getLogger(__name__)
+
+
+NIKO_HOME_CONTROL_THERMOSTAT_MODES_MAP = {
+    "off": 3,
+    "cool": 4,
+    "auto": 5,
+}
+
+
+class NIKO_HOME_CONTROL_THERMOSTAT_MODES(int):
+    """Niko Home Control thermostat modes."""
+
+    OFF = 3
+    COOL = 4
+    AUTO = 5
```

Comment:
whats this for

Reply:
Just to make it a little bit more readable but again something that should live in the lib.